dev_tools/scripts/scw

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env ruby
+
+# frozen_string_literal: true
+
+if `command -v sem`.empty?
+  abort <<~ERROR
+    sem CLI not found
+
+    brew install semaphoreci/tap/sem
+
+    docs: https://docs.semaphoreci.com/reference/sem-command-line-tool/
+  ERROR
+end
+
+branch = `git branch --show-current`.strip
+
+workflows = `sem get workflows`.split("\n")
+
+begin
+  workflow_row = workflows.find { |workflow| workflow.match?(branch) }
+  workflow_id = workflow_row.split.first
+rescue NoMethodError
+  abort 'workflow not found'
```

Comment:
Consistency
```suggestion
  abort "workflow not found"
```

Reply:
I actually prefer the default Rubocop setting for this which is what's applied since the script lives outside the other repos.

Reply:
That's fine... it was just the inconsistency I'd noticed :)